### PR TITLE
fix: add pool find button to v2 pool page

### PR DESCRIPTION
**Pull request by ianlapham**
- add buttont for importing v2 pool 

fixes https://github.com/Uniswap/uniswap-interface/issues/1696

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default function App() {
                 <Route exact strict path="/swap/:outputCurrency" component={RedirectToSwap} />
                 <Route exact strict path="/swap" component={Swap} />
 
-                <Route exact strict path="/find" component={PoolFinder} />
+                <Route exact strict path="/pool/v2/find" component={PoolFinder} />
```
Comment (justindomingue) [inline src/pages/App.tsx:102]: why not keep /find?

File: src/pages/PoolFinder/index.tsx [hunk 2]
```diff
@@ -81,19 +81,19 @@ export default function PoolFinder() {
   const prerequisiteMessage = (
     <LightCard padding="45px 10px">
       <Text textAlign="center">
-        {!account ? 'Connect to a wallet to find pools' : 'Select a token to find your liquidity.'}
+        {!account ? 'Connect to a wallet to find pools' : 'Select a token to find your v2 liquidity.'}
```
Comment (justindomingue) [inline src/pages/PoolFinder/index.tsx:84]: nit. uppercase 'V2' for consistency

File: src/pages/App.tsx (hunk 1 above)
Comment (justindomingue) [reply src/pages/App.tsx:102]: atcually.. it probably makes sense to use pool/v2/find

**Review by moodysalem — APPROVED** [review]

**Comment by moodysalem** [comment]
i just noticed we have a link from https://app.uniswap.org/#/migrate/v2 that needs to be fixed

**Comment by willhennessy** [comment]
@ianlapham can we merge and ship? It's been a week since the tweet reported the issue

**Pull request merged**